src/test/java/com/fasterxml/jackson/core/testutil/failure/ExpectedPassingTestCasePredicate.java

```diff
@@ -0,0 +1,15 @@
+package com.fasterxml.jackson.core.testutil.failure;
+
+import java.util.List;
+
+/**
+ * Interface that defines a predicate to determine if a test case is expected to pass.
```

Comment:
I assume "pass" here mean that test completes without exception? (if so, fine, if not should probably be explained further).